gravitee-apim-portal-webui-next/src/scss/theme/variables.scss

```
@@ -60,6 +60,8 @@ $nav-bar-link-active-background-color: var(
 );
 $nav-bar-border-color: var(--gio-app-nav-bar-outline-color, $border-color);
 $nav-bar-elevation: var(--gio-app-nav-bar-elevation);
+$nav-bar-mobile-menu-icon-color: var(--gio-app-nav-mobile-menu-icon-color, #000);
+$nav-bar-mobile-menu-box-shadow: var(--nav-bar-mobile-menu-box-shadow, 0 11px 15px 0 rgb(0 0 0 / 20%));
```

```suggestion
$nav-bar-mobile-menu-box-shadow: var(--gio-app-nav-bar-mobile-menu-elevation-shadow, 0 11px 15px 0 rgb(0 0 0 / 20%));
```
Missing prefix and matching material vocabulary. WDYT?